clang/test/CodeGenDirectX/unsupported_intrinsic.hlsl

```diff
@@ -0,0 +1,7 @@
+// RUN: %if clang-dxc %{not %clang_dxc -T lib_6_3 %s 2>&1 | FileCheck %s %}
```

Comment:
I don't think there's a `clang-dxc` feature, so I suspect this just never runs at all. This will fail if the DirectX backend isn't available, so I don't think gating on clang-dxc would be correct anyway.

I think you actually want `REQUIRES: directx-registered-target` in this test, and to drop the `%if` stuff.

Reply:
I tried REQUIRES: directx-registered-target and it just makes it so the test gets skipped as unsupported even on my local machine with the directx target built